model/bootstrap/filenames.go

```diff
@@ -20,6 +20,7 @@ var (
 	PathNodeInfosPub          = filepath.Join(DirnamePublicBootstrap, "node-infos.pub.json")
 	PathPartnerNodeInfoPrefix = filepath.Join(DirnamePublicBootstrap, "node-info.pub.")
 	PathNodeInfoPub           = filepath.Join(DirnamePublicBootstrap, "node-info.pub.%v.json") // %v will be replaced by NodeID
+	PartnerStakesFileName     = "partner-stakes.json"
```

Comment:
```suggestion
	FilenamePartnerStakes     = "partner-stakes.json"
```
consistent naming with other filename constants

Reply:
https://github.com/onflow/flow-go/pull/1988/commits/93b09c2b5b78314c38c0cb496c64378b5e740376